src/components/cylc/gantt/GanttChart.vue

```diff
@@ -0,0 +1,219 @@
+
+        for (let i = 0; i < taskNameList.length; i++) {
+          taskColours[taskNameList[i]] = colours[i % colours.length]
+        }
+        const timingOptions = {
+          total: { start: 'submittedTime', end: 'finishedTime' },
+          run: { start: 'startedTime', end: 'finishedTime' },
+          queue: { start: 'submittedTime', end: 'startedTime' },
+        }
+        const { start, end } = timingOptions[this.timingOption]
+        for (let i = startTask; i < endTask; i++) {
+          for (let j = 0; j < jobs[taskNameList[i]].length; j++) {
+            data.push({
+              x: taskNameList[i],
+              y: [
+                new Date(jobs[taskNameList[i]][j][start]).getTime(),
+                new Date(jobs[taskNameList[i]][j][end]).getTime()
+              ],
+              fillColor: taskColours[taskNameList[i]],
+            })
+          }
+        }
+      }
+      return [{ data }]
+    },
+    numPages () {
+      if (this.jobs.length !== 0) {
+        return Math.ceil(Object.keys(this.jobs[0]).length / this.tasksPerPage)
+      } else {
+        return 1
+      }
+    },
+
+    chartOptions () {
+      return {
+        chart: {
+          animations: {
+            enabled: this.animate && !this.reducedAnimation,
+            easing: 'easeinout',
+            speed: 300,
+            animateGradually: {
+              enabled: true,
+              delay: 150,
+            },
+            dynamicAnimation: {
+              enabled: true,
+              speed: 350,
+            },
+          },
+          fontFamily: 'inherit',
+          toolbar: {
+            tools: {
+              download: `<svg class="w-100 h-100"><path d="${mdiDownload}"></path></svg>`,
+            },
+          },
+        },
+        tooltip: {
+          custom: function ({ seriesIndex, dataPointIndex, w }) {
+            const startTime = new Date(w.config.series[seriesIndex].data[dataPointIndex].y[0])
+            const finishTime = new Date(w.config.series[seriesIndex].data[dataPointIndex].y[1])
+            return (
+              '<div class="apexcharts-tooltip-candlestick">' +
+              '<div>Start: <span class="value">' +
+              startTime +
+              '</span></div>' +
+              '<div>Finish: <span class="value">' +
+              finishTime +
+              '</span></div>' +
+              '</div>'
+            )
+          },
+        },
+        plotOptions: {
+          bar: {
+            horizontal: true,
+          },
+        },
+        xaxis: {
+          labels: {
+            formatter: function (value, timestamp, opts) {
+              return new Date(value).toTimeString().slice(0, 9)
+            }
+          },
+          title: {
+            text: 'Time',
+          },
+        },
+        yaxis: {
+          labels: {
+            maxWidth: 280,
+            offsetX: -10,
+          },
+        },
+      }
+    },
+  },
+}
+</script>
+
+<style scoped>
+.apexcharts-text {
```

Comment:
Oh, it turns out you have to do
```suggestion
:deep(.apexcharts-text) {
```
to get this to apply within the `VueApexCharts` child component

https://vuejs.org/api/sfc-css-features.html#deep-selectors

Reply:
done